src/common/AdaptiveStream.cpp

```diff
@@ -230,7 +230,10 @@ bool AdaptiveStream::download(const std::string& url,
     bool isEOF{false};
     std::string transferEncodingStr{
         file.GetPropertyValue(ADDON_FILE_PROPERTY_RESPONSE_HEADER, "Transfer-Encoding")};
-    bool isChunked{transferEncodingStr.find("hunked") != std::string::npos};
+    std::string contentLengthStr{
+            file.GetPropertyValue(ADDON_FILE_PROPERTY_RESPONSE_HEADER, "Content-Length") };
+    bool isChunked{contentLengthStr.empty() ||
```

Comment:
Can you add a little comment that explain why has been done in this way?
something like:
HTTP2 content is always chunked and Content-Length header should not be provided

Reply:
please reapply clang

Reply:
Thanks, I have added some detailed comments